### ISSUE TITLE
INFO: Lower confidence scores with supervised_embeddings since 1.3

### ISSUE THREAD

**Issue by akelad** [RasaHQ/rasa#4857]
Since version 1.3 (with the supervised_embeddings pipeline) you may have been seeing some very low confidence scores even though other intents don't have very high confidence, like this one for example:
```
how have you been
{
  "intent": {
    "name": "ask_howdoing",
    "confidence": 0.6736818552017212
  },
  "entities": [],
  "intent_ranking": [
    {
      "name": "ask_howdoing",
      "confidence": 0.6736818552017212
    },
    {
      "name": "ask_whoisit",
      "confidence": 0.06416542083024979
    }
```
The reason for this is that with our new default architecture uses a softmax loss function. This means each intent will have some sort of probability associated with it, so if you have 50 intents and all the "least confident" intents have a confidence of `0.1`, they will already take up `49*0.1 = 0.49` of the confidence level and so your top intent will only have a confidence of `0.5`. The more intents you have, the lower the confidence level will be. 

Some temporary solutions for this:
- lower your nlu threshold in your fallback policy
- change the `loss_type` to `margin` in your [policy configuration](https://rasa.com/docs/rasa/nlu/components/#embeddingintentclassifier)

We are working on changing this behaviour soon by adding some form of normalisation to the scores. https://github.com/RasaHQ/research/issues/57

**Comment by akelad** [comment]
as of release 1.7.0 probabilities can be normalised across the top n intents: https://rasa.com/docs/rasa/migration-guide/#rasa-1-6-to-rasa-1-7
